mathics/builtin/base.py

```diff
@@ -190,13 +190,25 @@ def get_functions(self, prefix='apply'):
                 else:
                     yield (pattern, function)
 
-    def get_option(self, options, name, evaluation, pop=False):
-        name = ensure_context(name)
-        value = options.pop(name, None) if pop else options.get(name)
-        if value is not None:
-            return value.evaluate(evaluation)
-        else:
-            return None
+    @staticmethod
+    def get_option(options, name, evaluation, pop=False):
+        # we do not care, whether an option X is given as
+        # System`X, Global`X, or "X". we always handle it
+        # as the same option. this matches Wolfram Language
+        # behaviour.
+
+        for variant in ('System`%s', 'Global`%s', '"%s"'):
```

Comment:
Should we look at all the entries in `$ContextPath` or even just drop the context entirely?